mcs/build/platforms/darwin.make

```diff
@@ -9,9 +9,9 @@ PLATFORM_RUNTIME = $(RUNTIME)
 PLATFORM_CORLIB = mscorlib.dll
 PLATFORM_TEST_HARNESS_EXCLUDES = NotOnMac,
 
-EXTERNAL_MCS = mcs
-EXTERNAL_MBAS = mbas
-EXTERNAL_RUNTIME = mono
+EXTERNAL_MCS := $(shell which mcs)
```

Comment:
In fact most of our Makefile variables could use `:=`, not `=`, and this would probably improve build performance. A separate PR is probably in order.